src/test/resources/json_files/admin_user_integration/publisher/filter_publishers_admin_view_response.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "code" : "PUBLISHER_A",
+    "name" : "Publisher A",
+    "status" : "ACTIVE",
+    "token" : {"type": "API_TOKEN"},
+    "contact":{}
+  }
+]
```

Comment:
Does the server response include the widgetLocation? Shall we not include is here as well?